jsonnet/components/control-plane.libsonnet

```diff
@@ -93,7 +93,7 @@ function(params)
                 action: 'replace',
                 targetLabel: '__address__',
                 regex: '(.+)(?::\\d+)',
-                replacement: '$1:9537',
+                replacement: 'https://$1:9637',
```

Comment:
you want to keep this line as-is because `__address__` is only `<address>:<port>`

what you need is to add `scheme: https` to the endpoint definition:

```
          // Collect metrics from CRI-O.
          [{
            interval: '30s',
            port: 'https-metrics',
            scheme: 'https',

```